docs/api/walkthrough-testing-the-backend-services-workflow.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: default
+title: "Walkthrough: Testing the Backend Services Workflow"
+parent: API
+nav_order: 5
+---
+This page documents a set of steps to connect to the Sara Alert FHIR API using the SMART on FHIR Backend Services Workflow as described in  [Getting Started](api-getting-started#smart-on-fhir-backend-services-workflow).
+
+**If you are testing against a local instance of Sara Alert, follow all of the instructions below. If you are testing against the demo server, ignore sections with (LOCAL TESTING ONLY), and note that some numbers in the list of instructions will be skipped.**
+
+## Setup the Environment (LOCAL TESTING ONLY)
+If you are testing on the demo server, skip to the next section.
+<details>
+<summary>Expand only if testing on a LOCAL instance of Sara Alert</summary>
+<div markdown="1">
+
+**1.** Clone and run Sara Alert following the steps in the [README](https://github.com/SaraAlert/SaraAlert/blob/master/README.md) for local setup. Make sure to have the database, Redis, and Sidekiq running for the full experience. At a minimum, the database and Redis need to be running.
+
+**2.** Optionally, connect to the database to query some of the tables as we go through the workflow using `mysql --user=disease_trakker`
+</div>
+</details>
+
+## Create a JSON Web Key (JWK)
+
+**3.** For this tutorial, use https://mkjwk.org. Create a JWK with the following settings: Key Size `2048`, Key Use `Signature`, Algorithm `RS384`, Key ID `SHA-256`, Show X.509 `Yes`. Click the "Generate" button.
```

Comment:
Here too, the https://mkjwk.org is not formatting as a clickable link.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/598/commits/490bef24961bf80cb728c31cd3185f0d063a4d0b